test/ServiceBus.Tests/Streaming/EHStreamProviderCheckpointTests.cs

```diff
@@ -171,7 +171,7 @@ private static void AdjustConfig(ClusterConfiguration config)
         {
             // register stream provider
             config.Globals.RegisterStreamProvider<EventHubStreamProvider>(StreamProviderName, BuildProviderSettings());
-            config.AddAzureTableStorageProvider(ImplicitSubscription_RecoverableStream_CollectorGrain.StorageProviderName);
+            config.AddAzureBlobStorageProvider(ImplicitSubscription_RecoverableStream_CollectorGrain.StorageProviderName);
```

Comment:
you just changed to use Blob storage..

Reply:
Yep.  I did this because these tests don't really care about what storage they use, and it was easier than adding a builder just to use azure table storage.  We'll eventually need to add a builder when we move other providers to the new model, but I didn't see the need to address that in this PR.

Anyway, I think we need more tests to use blob storage anyway.  Most of our tests use table storage.